include/real/real_data.hpp

```diff
@@ -337,6 +338,99 @@ namespace boost {
                     break;
                 }
 
+                case OPERATION::EXPONENT :{
+                    this->_approximation_interval.lower_bound = 
+                        exponent(ro.get_lhs_itr().get_interval().lower_bound.up_to(_precision, false), _precision, false);
```

Comment:
What happens when input interval includes zero?

Reply:
Exponent is defined for all numbers, no problem with that. For all other functions, I have made proper checks for these types of situations. And, also exponent is always increasing functions, so no other checks required.